shared/rangeanalysis/codeql/rangeanalysis/RangeAnalysis.qll

```diff
@@ -296,11 +373,24 @@ signature module LangSig<Semantic Sem, DeltaSig D> {
 }
 
 signature module BoundSig<LocationSig Location, Semantic Sem, DeltaSig D> {
+  /**
+   * A semantic bound, which defines a constraint on the possible values of an
+   * expression.
```

Comment:
I was planning to ditch all of those `Sem` prefixes eventually, so let's not copy "semantic" into the qldoc. In any case, here's how I'd describe this:
```suggestion
   * A bound that the range analysis can infer for a variable. This includes
   * constant bounds represented by the abstract value zero, SSA bounds for when
   * a variable is bounded by the value of a different variable, and possibly
   * other abstract values that may be useful variable bounds. Since all bounds
   * are combined with an integer delta there's no need to represent constant
   * bounds other than zero.
```

Reply:
Thanks for these suggestions.